parm/snow/obs/config/sfcsno_snow.yaml.j2

```diff
@@ -5,9 +5,11 @@
       halo size: 250e3
     obsdatain:
       engine:
-        type: bufr
+        type: script
+        script file: "{{ USHgfs }}/bufr2ioda_sfcsno_bufr_encoder.py"
+        args:
+          input_path: '{{ DATA }}/obs/{{ OPREFIX }}sfcsno.tm00.bufr_d'
```

Comment:
Suggestion:
Make the mapping file as input to the Python script for `create_obs_group`
  ```
args:
   input_path:  '{{ DATA }}/obs/{{ OPREFIX }}sfcsno.tm00.bufr_d'
   mapping_path: = "./obs/bufr_sfcsno_mapping.yaml"
```

This will make it clear that the script backend requires two inputs:  bufr file and the mapping file

Reply:
Thanks @emilyhcliu for the suggestions. I made the changes as suggested.